poly-synth/src/poly-synth-demo.js

```diff
@@ -76,7 +176,7 @@ class PolySynthDemo {
    * @param {Number} value The new gain.
    */
   setGain(value) {
-    this.masterGain_.gain.value = parseFloat(value);
+    this.masterGain_.gain.value = value;
```

Comment:
Using `parseFloat` was correct so long as lib/param-controller.js did not to string to number conversion. I think that change to lib/param-controller.js got landed before the poly-synth, so, yes this was a mistake that we overlooked.